common/http_test.go

```diff
@@ -311,6 +311,18 @@ func TestUnmarshall(t *testing.T) {
 	require.NoError(t, err)
 }
 
+func TestUnmarshallError(t *testing.T) {
+	v := struct {
+		Correct   string `json:"c"`
+		Incorrect int    `json:"i"`
+	}{}
+	ws := DatabricksClient{}
+	err := ws.unmarshall("/a/b/c", []byte(`{"c":"val", "i":"123"}`), &v)
+	require.Error(t, err)
+	require.EqualError(t, err,
+		`Invalid JSON: json: cannot unmarshal string into Go struct field .i of type int - {"c":"val", "i":"123"}`)
```

Comment:
in 99.999% of cases there'll be `Invalid JSON: json: cannot`, so that's why i suggest to make the format as simple as `%s\n%s`